homeassistant/components/overkiz/climate_entities/atlantic_pass_apc_heating_and_cooling_zone.py

```diff
@@ -0,0 +1,134 @@
+"""Support for Atlantic Pass APC Heating And Cooling Zone Control."""
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+from pyoverkiz.models import Command
+
+from homeassistant.components.climate import (
+    PRESET_AWAY,
+    PRESET_COMFORT,
+    PRESET_ECO,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACMode,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+from ..entity import OverkizEntity
+
+PRESET_FROST_PROTECTION = "frost_protection"
+
+OVERKIZ_TO_HVAC_MODE: dict[str, str] = {
+    OverkizCommandParam.AUTO: HVACMode.AUTO,
+    OverkizCommandParam.ECO: HVACMode.AUTO,
+    OverkizCommandParam.MANU: HVACMode.HEAT,
+    OverkizCommandParam.HEATING: HVACMode.HEAT,
+    OverkizCommandParam.STOP: HVACMode.OFF,
+    OverkizCommandParam.INTERNAL_SCHEDULING: HVACMode.AUTO,
+    OverkizCommandParam.COMFORT: HVACMode.HEAT,
+}
+
+HVAC_MODE_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_HVAC_MODE.items()}
+
+
+OVERKIZ_TO_PRESET_MODES: dict[str, str] = {
+    OverkizCommandParam.OFF: PRESET_ECO,
+    OverkizCommandParam.STOP: PRESET_ECO,
+    OverkizCommandParam.COMFORT: PRESET_COMFORT,
+    OverkizCommandParam.MANU: PRESET_COMFORT,
+    OverkizCommandParam.ABSENCE: PRESET_AWAY,
+    OverkizCommandParam.ECO: PRESET_ECO,
+    OverkizCommandParam.INTERNAL_SCHEDULING: PRESET_COMFORT,
+}
+
+PRESET_MODES_TO_OVERKIZ = {v: k for k, v in OVERKIZ_TO_PRESET_MODES.items()}
+
+THERMAL_CONFIGURATION_STATE = "core:ThermalConfigurationState"
```

Comment:
Could you do a PR to https://github.com/iMicknl/python-overkiz-api to add this as an enum?

Reply:
Done: https://github.com/iMicknl/python-overkiz-api/pull/607